cmd/main.go

```diff
@@ -53,11 +67,15 @@ var (
 	leaderElectionRetryPeriod   = flag.Duration("leader-election-retry-period", 5*time.Second, "Duration, in seconds, the LeaderElector clients should wait between tries of actions. Defaults to 5 seconds.")
 	lockReleaseSyncPeriod       = flag.Duration("lock-release-sync-period", 60*time.Second, "Duration, in seconds, the sync period of the lock release controller. Defaults to 60 seconds.")
 
+	enableStateful = flag.Bool("enable-stateful-multishare", false, "")
```

Comment:
we are following a standard prefix "feature-". e.g https://github.com/kubernetes-sigs/gcp-filestore-csi-driver/blob/master/cmd/main.go#L49 for feature-lock-release

Reply:
fixed in most recent code version. will be included in upcoming PR